docs/source/development/automated_testing.md

```diff
@@ -0,0 +1,165 @@
+
+```
+============================= test session starts ==============================
+...
+collected 1 item
+
+src/tests/test_run.py .                                                  [100%]
+
+============================== 1 passed in 0.38s ===============================
+```
+
+This output indicates that one test ran successfully in the file `src/tests/test_run.py`.
+
+## Add test coverage reports with `pytest-cov`
+
+It can be useful to see how much of your project is covered by tests. For this, you can install and configure the [`pytest-cov`](https://pypi.org/project/pytest-cov/) plugin for `pytest`, which is based on the popular [`coverage.py`](https://coverage.readthedocs.io/) library.
+
+### Install `pytest-cov`
+
+Install `pytest` as you would install other packages with pip, making sure your project's virtual environment is active.
+
+```bash
+pip install pytest-cov
+```
+
+### Configure `pytest` to use `pytest-cov`
+
+To configure `pytest` to generate a coverage report using `pytest-cov`, you can add the following lines to your `<project_root>/pyproject.toml` file (creating it if it does not exist).
+
+```
+[tool.pytest.ini_options]
+addopts = """
+--cov-report term-missing \
+--cov src/<project_name> -ra"""
+```
+
+### Run `pytest` with `pytest-cov`
+
+Running `pytest` in the spaceflights starter with `pytest-cov` installed results in the following additional report.
+
+```
+Name                                                     Stmts   Miss  Cover   Missing
+--------------------------------------------------------------------------------------
+src/spaceflights/__init__.py                                 1      1     0%   4
+src/spaceflights/__main__.py                                30     30     0%   4-47
+src/spaceflights/pipeline_registry.py                        7      7     0%   2-16
+src/spaceflights/pipelines/__init__.py                       0      0   100%
+src/spaceflights/pipelines/data_processing/__init__.py       1      1     0%   3
+src/spaceflights/pipelines/data_processing/nodes.py         25     25     0%   1-67
+src/spaceflights/pipelines/data_processing/pipeline.py       5      5     0%   1-8
+src/spaceflights/pipelines/data_science/__init__.py          1      1     0%   3
+src/spaceflights/pipelines/data_science/nodes.py            20     20     0%   1-55
+src/spaceflights/pipelines/data_science/pipeline.py          8      8     0%   1-40
+src/spaceflights/settings.py                                 0      0   100%
+--------------------------------------------------------------------------------------
+TOTAL                                                       98     98     0%
+```
+
+This is the simplest report that `coverage.py` (via `pytest-cov`) will produce. It gives an overview of how many of the executable statements in each project file are covered by tests. For a full set of features offered by `coverage.py`, see the [docs](https://coverage.readthedocs.io/).
```

Comment:
```suggestion
This is the simplest report that `coverage.py` (via `pytest-cov`) will produce. It gives an overview of how many of the executable statements in each project file are covered by tests. For a full set of features offered by `coverage.py`, see the [coverage docs](https://coverage.readthedocs.io/).
```

Reply:
Fixed in [8650a0b](https://github.com/kedro-org/kedro/pull/1879/commits/8650a0bd5726382b78f1eeaa7439d121ac297ae3).